testing/regression/cypress/e2e/pages/events/add-lab-report-investigation.page.js

```diff
@@ -0,0 +1,74 @@
+ class AddLabReportInvestigation {
```

Comment:
Accidentally included this but it's a good function used for E2e LabReport and Investigation creation.